Common/GPU/OpenGL/GLFeatures.cpp

```diff
@@ -626,11 +626,11 @@ bool CheckGLExtensions() {
 }
 
 void SetGLCoreContext(bool flag) {
-	_assert_msg_(!extensionsDone, "SetGLCoreContext() after CheckGLExtensions()");
```

Comment:
I still think this should assert, at least if the flag is different?  It's confusing if this silently does nothing.  Better defensive programming for it to warn you that you've probably made a mistake.

-[Unknown]

Reply:
Yeah, you're right - this asserts now when you restart the context though, but asserting if the flag changes after extensions done would fix it.